bgpd/bgp_packet.c

```diff
@@ -1972,38 +1971,29 @@ static int bgp_route_refresh_receive(struct peer *peer, bgp_size_t size)
 					} else
 						p_pnt = p_end;
 
-					if ((ok = (p_pnt < p_end)))
-						orfp.ge =
-							*p_pnt++; /* value
-								     checked in
-								     prefix_bgp_orf_set()
-								     */
-					if ((ok = (p_pnt < p_end)))
-						orfp.le =
-							*p_pnt++; /* value
-								     checked in
-								     prefix_bgp_orf_set()
-								     */
+					// value checked in prefix_bgp_orf_set
+					if (p_pnt < p_end)
+						orfp.ge = *p_pnt++;
+
+					// value checked in prefix_bgp_orf_set
+					if (p_pnt < p_end)
+						orfp.le = *p_pnt++;
+
 					if ((ok = (p_pnt < p_end)))
 						orfp.p.prefixlen = *p_pnt++;
-					orfp.p.family = afi2family(
-						afi); /* afi checked already  */
-
-					psize = PSIZE(
-						orfp.p.prefixlen); /* 0 if not
-								      ok */
-					if (psize
-					    > prefix_blen(
-						      &orfp.p)) /* valid for
-								   family ?   */
-					{
+
+					// afi checked already
```

Comment:
nit: c++ -style comment - is that a thing now?

Reply:
please feel free to grep the code base.  They are all over it.

Reply:
interesting - `checkpatch` complains if I try to add a comment like this?

Reply:
@donaldsharp mark is correct, C style comments are the standard. I would have removed // by now but clang-format does not do comment processing.